certlint_test.go

```diff
@@ -46,37 +51,166 @@ accZ5e1iOtNIBTVUEaQ0YxNwhZqnQO4rgBFCnEpNH6TB62hXUD6u/oP/WB48Wmek
 KPgaoWHMo5sOFQnw5A==
 -----END CERTIFICATE-----`
 
+func TestMain(m *testing.M) {
+	build := exec.Command("go", "build")
+	berr := build.Run()
+	if berr != nil {
+		fmt.Printf("could not build before tests: %v", berr)
+		os.Exit(1)
+	}
+
+	retcode := m.Run()
+
+	clean := exec.Command("go", "clean", "-x")
```

Comment:
Do you think `go clean` is required here? Normally `go` would take care of this automatically not?

Reply:
True that relying on Travis would mean that one would have to perform "go install" or at the very least "go build" before being able to run local CLI tests.

Unfortunately, when using TestMain to set things up for the test, one must also clean up. Without `go clean`, the binary is left hanging around in the base directory.